snapshots/wash/2024-02-15/nutrients_europe.py

```diff
@@ -0,0 +1,25 @@
+"""Script to create a snapshot of dataset."""
```

Comment:
You can add some general instructions on how to download the file and upload it manually